### ISSUE TITLE
Won't work on iOS 10.0.2

### ISSUE THREAD

**Issue by xxAndreixx** [EddyVerbruggen/cordova-plugin-safariviewcontroller#44]
Hi, thank you for your work.

I'm trying to open links using your plugin but it won't work on iOS. Debugging it wouldn't show any error.

I installed the plugin, and I'm using your exact example, but it won't open anything on an iPad Pro with iOS 10.0.2.

**Comment by EddyVerbruggen** [comment]
Dunno, perhaps something whitelisting related? Google it for a bit..

**Comment by xxAndreixx** [comment]
Yes, sorry about not giving you more details. I'm already using the whitelist plugin and I do whitelist everything, all origin access just in case. And also I trace it with alerts all over and the function will fire without a problem, but it won't run anything. I've just updated to the latest version of iOS and I thought also it might be a problem with them, but I wasn't able to find anything related to it. I'll keep digging. Also, if I may, does this plugin work to open links like: http://www.maydomain.com/name_of_pdf.pdf ? Will it be shown in Safari?

**Comment by EddyVerbruggen** [comment]
I guess if it opens in regular Safari it will
